routers/userRouting.js

```diff
@@ -0,0 +1,52 @@
+const Router = require('koa-router');
+const router = new Router();
+const db = require('../services/userData');
+
+router.get('/', async ctx => {
+  return 'HomePage';
+});
+
+router.get('/users', async ctx => {
+  return db.getAll();
+});
+
+router.get(`/user/:username`, async ctx => {
+  const { username } = ctx.params;
+  return db.getOne(username);
+});
+
+router.post('/register', async ctx => {
+  const bodyParsed = ctx.request.body;
+  const data = bodyParsed;
+  const username = data.username;
+  const password = data.password;
+  const email = data.email;
+  return db.createOne(username, password, email);
+});
+
+router.post('/login', async ctx => {
+  const bodyParsed = ctx.request.body;
+  const data = bodyParsed;
+  const username = data.username;
+  const password = data.password;
+  return db.login(username, password);
```

Comment:
use `{username, password}` syntax for you functions

Reply:
`router.post('/register', async ctx => {
  const bodyParsed = ctx.request.body;
  const data = bodyParsed;
  return db.createOne(data);
});`  I'm transferring data object, where are all we got from JSON and in our createOne function we use information from our transferred object.